docs/src/single-validator-manager.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Single Validator Stake Pool Manager Program
+---
+
+A work-in-progress program for permissionless management of stake pools that are
+only delegated to one validator.
+
+## Operation
+
+The single-validator manager program defines a canonical stake pool for any
+validator on the network, which can be created by anyone.
+
+The program manages the "delegation strategy" of the pool, which is to maximize
+the amount of stake on one validator, while maintaining a small amount of SOL
+in the reserve for small stakers.
+
+### Create Pool
+
+The `CreateStakePool` instruction allows anyone to create the canonical stake
+pool for a validator, at the cost of ~0.0124 SOL for account rent-exemption of
+the stake pool, validator list, reserve, mint, and fee account.
+
+After the pool is created, the reserve requires at least stake rent-exemption plus
+the minimum delegation amount to actually add the validator to the pool, which
+costs 1.0028288 SOL. The user receives pool tokens for this deposit.
+
+After the SOL deposit, the `AddValidatorToPool` instruction on the pool management
```

Comment:
This instruction does create a stake account from scratch on a PDA, with a lockup configured by the pool. Since all pools have a default lockup (none), the stake account won't have a lockup either.